### DISCUSSION TITLE
The Granny smith apple has its origins where?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#498] [Galaxy Trivia] [answered]
Named for its discoverer, Maria Ann Smith 🍏

**Comment by thuongtruong1009** [comment, marked as answer]
The Granny Smith apple originated in Australia, specifically in the city of Eastwood, located in the state of New South Wales. The apple was first grown by Maria Ann Smith, also known as Granny Smith, in the mid-19th century. The variety is named after her and is known for its bright green skin, tart flavor, and firm texture, which makes it a popular choice for cooking and baking.
